package.json

```diff
@@ -47,8 +47,8 @@
     "access": "public"
   },
   "dependencies": {
-    "@sasjs/adapter": "3.7.4",
-    "@sasjs/core": "4.9.2",
+    "@sasjs/adapter": "file:../adapter/build/sasjs-adapter-5.0.0.tgz",
```

Comment:
Adapter shouldn't be installed from a tarball

Reply:
Once, new adapter version is published, I may update this.

Reply:
Updated!